cli/src/config/templates.toml

```diff
@@ -243,6 +238,20 @@ if(commit.hidden(),
 )
 '''
 
+'format_commit_header(commit)' = '''
```

Comment:
nit: rename to `format_compact_commit_header` or `format_short_commit_header`?

If we split `builtin_log_detailed`, the header part would be called "format_detailed_commit_header" or something.